auth/src/main/java/com/firebase/ui/auth/util/signincontainer/SmartLockBase.java

```diff
@@ -21,6 +21,7 @@
 import com.google.firebase.auth.FacebookAuthProvider;
 import com.google.firebase.auth.FirebaseUser;
 import com.google.firebase.auth.GoogleAuthProvider;
+import com.google.firebase.auth.PhoneAuthProvider;
```

Comment:
This doesn't look necessary since there are no other changes in this class. Should there be some though? Wouldn't we want to update and use `providerIdToAccountType` somewhere?